go/client/cmd_sigs_revoke.go

```diff
@@ -55,7 +55,8 @@ func (c *CmdSigsRevoke) Run() error {
 func NewCmdSigsRevoke(cl *libcmdline.CommandLine) cli.Command {
 	return cli.Command{
 		Name:         "revoke",
-		ArgumentHelp: "<id>",
+		ArgumentHelp: "<id> ...",
+		Usage:        "Verify one or more signatures by sig ID",
```

Comment:
Verify?  That doesn't sound right.

Also, I believe there was no Usage for this command so it wouldn't show up in help.  We can tell users about it if they need it, but don't want them running it without guidance.